app/views/claims/email_address.html.erb

```diff
@@ -9,35 +9,29 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render("shared/error_summary", instance: @form) if @form.errors.any? %>
+    <%= form_for @form, url: claim_path(current_journey_routing_name), builder: GOVUKDesignSystemFormBuilder::FormBuilder, html: { novalidate: false } do |f| %>
+      <%= f.govuk_error_summary %>
 
-    <%= form_for @form, url: claim_path(current_journey_routing_name), html: { novalidate: false } do |f| %>
-      <span class="govuk-caption-xl">
-        <%= I18n.t("questions.personal_details") %>
-      </span>
-
-      <%= form_group_tag f.object do %>
-        <h1 class="govuk-label-wrapper">
-          <%= f.label :email_address, t("questions.email_address"), class: "govuk-label #{label_css_class_for_journey(f.object.journey)}" %>
-        </h1>
-
-        <div id="email-address-hint" class="govuk-hint">
+      <%= f.govuk_text_field :email_address,
+        autocomplete: "email",
+        spellcheck: "false",
+        caption: {
+          text: I18n.t("questions.personal_details"),
+          size: "xl"
+        },
+        label: {
+          text: "Email address",
```

Comment:
Should this be
```suggestion
          text: t("questions.email_address")
```

(though I prefer plain text, i18n is more trouble than it's worth!)

Reply:
I've just left these as is from before, but yes we can probably just start hard coding these

Reply:
Wait, I misunderstood, yeah i can change this. I'm not sure if form builder uses I18n out the box it would be could to not have to set these if formbuilder reads the correct location in the i18n values